src/types/config.ts

```diff
@@ -14,11 +14,43 @@ export type OverrideJiterConfigOptions = {
   millisecondsUntilWebhookExpiration?: number;
 };
 
+export type EncryptionOptions<Key = string> = {
+  /**
+   * The encryption keys used to encrypt and decrypt data before sending it to Jiter.
+   *
+   * The last key in the array is the key used to encrypt new data. Old keys are used to decrypt data that was encrypted with that key.
+   */
+  keys: Array<{
```

Comment:
Did you do this to avoid making a new type here? I'm ok with this, just think it's a little less readable

Reply:
Yeah, I didn't really know how you wanted this file, I was _very_ confused with all the types in this file and how they are supposed to function.